audits/README.md

```diff
@@ -5,48 +5,24 @@ The related code audits are provided for context.
 
 ## Audits:
 
-1. [Halborn Audit - 29.02.2021](./halborn/audit01-assets-bondedFinance-dutchAuction-oracle-vesting-dexRouter-mosaic.pdf)
-
-    Pallets:
-   - Assets
-   - Bonded Finance
-   - Dutch Auction
-   - Oracle
-   - Vesting
-   - Dex-Router
-   - Mosaic
+[Halborn Audit - Main Pallets - 29.02.2021](./halborn/audit20210229-pallets-core.pdf)
    
-2. [Halborn Audit - 30.07.2022](./halborn/audit03-pallets-pabloV2.pdf)
+[Halborn Audit - Pablo - 30.07.2022](./halborn/audit20220730-pallets-pabloV2.pdf)
 
-    Pallets:
-   - Pablo
+[Halborn Audit - Vesting - 23.08.2022](./halborn/audit20220823-pallet-vesting.pdf)
 
-3. [Halborn Audit - 23.08.2022](./halborn/audit04-vesting-pallet.pdf)
+[Halborn Audit - Democracy - 19.09.2022](./halborn/audit20220919-pallet-democracy.pdf)
 
-    Pallets: 
-   - Vesting
+[Halborn Audit - BYOG - 26.09.2022](./halborn/audit20220926-pallet-byog.pdf)
 
-4. [Halborn Audit - 19.09.2022](./halborn/audit05-democracy-Pallet.pdf)
+[Halborn Audit - Grandpa Light Client Bridge 03.10.2022](./halborn/audit20221003-pallet-grandpa-light-client-bridge.pdf)
 
-    Pallets: 
-   - Democracy
+[Halborn Audit - IBC - 10.10.2022](./halborn/audit20221010-pallet-ibc.pdf)
 
-5. [Halborn Audit - 26.09.2022](./halborn/audit06-byoGas.pdf)
+[Halborn Audit - Staking Rewards & fNFT- 10.10.2022](./halborn/audit20221010-pallets-stakingRewards-fnft.pdf)
 
-    Pallets:
-   - BYOG (Bring your own Gas)
+[Halborn Audit - Pablo Refactoring 29.11.2022](./halborn/audit20221129-pallet-pablo-refactoring.pdf)
 
-6. [Halborn Audit - 10.10.2022](./halborn/audit07-ibc-pallet.pdf)
-
-   Pallet:
-   - IBC (Inter-Blockchain-Communication)
-
-7. [Halborn Audit - 10.10.2022](./halborn/audit08-pallets-stakingRewards-fnft.pdf)
-
-   Pallets:
-   - Staking Rewards
-   - fNFT
-   
 ## Post-Mortems
 
```

Comment:
no need to put this here, and again, keep everything that has been audited in here